src/common/types/cpuArchitecture.ts

```diff
@@ -12,27 +13,19 @@ export enum CpuArchitecture {
   s390x = 's390x',
 }
 export enum OcmCpuArchitecture {
-  x86 = 'amd64',
+  x86 = 'x86_64',
```

Comment:
Why this change?
When this was implemented, we saw that from the OCM cluster we were receiving `amd64` as the CPU architecture. 

I looked for an example, and this cluster is from 1 month ago and it still has `amd64`.
So either it is `amd64` in all cases, or we may receive the two values, in which case we need to handle that.

![amd64](https://user-images.githubusercontent.com/829045/228172839-f6c19f7d-59f1-47d9-85a8-f50b3f3054d9.png)

Reply:
It's a mistake. Thanks for finding it!